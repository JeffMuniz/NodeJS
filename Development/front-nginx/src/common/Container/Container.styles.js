import styled from "styled-components";
import { Grid } from "react-styled-flexboxgrid";
import { white } from "../../styles/colors";

export const Wrapper = styled(Grid)`
  margin: 0 auto;
  background-color: ${white};
  border-radius: 6px;
`;
