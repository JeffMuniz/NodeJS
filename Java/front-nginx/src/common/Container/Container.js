import React from "react";
import PropTypes from "prop-types";
import { Wrapper } from "./Container.styles";

const Container = props => <Wrapper {...props} />;

export default Container;

Container.propTypes = {
  children: PropTypes.node.isRequired,
};
