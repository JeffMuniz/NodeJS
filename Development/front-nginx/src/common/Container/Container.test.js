import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import Container from "./Container";

describe("Container", () => {
  it("Should render Container", () => {
    // given
    const renderer = new ShallowRenderer();

    // when
    renderer.render(
      <Container>
        <div id="test" />
      </Container>,
    );
    const result = renderer.getRenderOutput();

    // then
    expect(result).toMatchSnapshot();
  });
});
