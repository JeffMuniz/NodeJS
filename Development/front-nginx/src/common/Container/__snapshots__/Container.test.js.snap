// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Should render Container 1`] = `
<Styled(Grid)>
  <div
    id="test"
  />
</Styled(Grid)>
`;
